src/glyph_cache.rs

```diff
@@ -63,8 +63,9 @@ pub enum SubpixelBin {
 
 impl SubpixelBin {
     pub fn new(pos: f32) -> (i32, Self) {
-        let (fract, truncf) = libm::modff(pos);
-        let trunc = truncf as i32;
+        let trunc = pos as i32;
+        let fract = pos - trunc as f32;
```

Comment:
Looking inside the implementation of `modff` reveals that it mostly just comes down to this here anyway as well as a bunch of special case handling (floats that are too large, too small, ...). I don't think those cases really matter here.